pkg/aggregator/aggregator.go

```diff
@@ -384,6 +398,30 @@ func (agg *BufferedAggregator) handleSenderBucket(checkBucket senderHistogramBuc
 	}
 }
 
+func (agg *BufferedAggregator) handleEventPlatformEvent(event senderEventPlatformEvent) error {
+	agg.mu.Lock()
+	defer agg.mu.Unlock()
+	if agg.eventPlatformForwarder == nil {
+		return errors.New("event platform forwarder not initialized")
```

Comment:
Don't you think we should just drop it (+ telemetry about the drops)? Maybe with a sync.Once to output an error message once?

Reply:
> Don't you think we should just drop it (+ telemetry about the drops)?

That's basically what we're doing here. It gets dropped, the error is tracked in the telemetry, and the error message will be logged out once per flush. 

> Maybe with a sync.Once to output an error message once?

We could do that but the downside is that the message only appears once, which could make it hard to find. The way it's working now is that the error will be logged once per flush interval, which increases the chance that the user will notice it while not being overly spammy. Do you think it would be better to log only once?

Reply:
No, you are completely right, it's fine how it is right now 👍  I think I've done this comment too early in my review, it is wrong.